lib/auth_app_web/controllers/user_session_controller.ex

```diff
@@ -4,20 +4,64 @@ defmodule AuthAppWeb.UserSessionController do
   alias AuthApp.Accounts
   alias AuthAppWeb.UserAuth
 
-  def create(conn, %{"_action" => "registered"} = params) do
-    create(conn, params, "Account created successfully!")
+  def create(conn, %{"_action" => "confirmed"} = params) do
+    create(conn, params, "User confirmed successfully.")
   end
 
-  def create(conn, %{"_action" => "password-updated"} = params) do
+  def create(conn, %{"_action" => "registered", "user" => user_params}) do
```

Comment:
I don't think we need this here now, do we? We just send the email without changes to the session.